src/handycon/handhelds/aya_gen7.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+# This file is part of Handheld Game Console Controller System (HandyGCCS)
+# Copyright 2022-2023 Derek J. Clark <derekjohn.clark@gmail.com>
+
+import sys
+from evdev import InputDevice, InputEvent, UInput, ecodes as e, list_devices, ff
+
+from .. import constants as cons
+
+handycon = None
+
+def init_handheld(handheld_controller):
+    global handycon
+    handycon = handheld_controller
+    handycon.BUTTON_DELAY = 0.10
+    handycon.CAPTURE_CONTROLLER = True
+    handycon.CAPTURE_KEYBOARD = True
+    handycon.CAPTURE_POWER = True
+    handycon.GAMEPAD_ADDRESS = 'usb-0000:c4:00.3-4/input0'
```

Comment:
This appears to match the gen6 device. If we can enable all the scancodes I see no reason why we couldn't use gen 6 instead.